spec/Section 2 -- Language.md

```diff
@@ -1257,6 +1298,17 @@ Type : Name
 - {type} must not be {null}
 - Return {type}
 
+Type : Type ?
+
+- Let {nullableType} be the result of evaluating {Type}
+- Return {nullableType}
```

Comment:
This is really confusing; I think delete the `Type : Type` block above, and instead have:

```suggestion
Type : Name ?

- Let {name} be the string value of {Name}
- Let {type} be the type defined in the Schema named {name}
- {type} must not be {null}
- Return {type}

Type : ListType ?

- Let {itemType} be the result of evaluating {ListType}
- Let {type} be a List type where {itemType} is the contained type.
- Return {type}
```